pkg/graphdb/conn_sqlite3_dynamic_windows.go

```diff
@@ -0,0 +1,17 @@
+// +build cgo,windows,dynamicsqlite
```

Comment:
I would vote not to have this file at all.  too much build flags and complexity while we are still trying to figure this stuff out.